Subnet-Calculator/assets/app.css

```
@@ -833,6 +886,17 @@
             .result-row { border-bottom: 1px solid #ccc; }
             .badge { border: 1px solid #999; }
             .split-list { columns: 2; display: block; }
+
+            /* #193: VLSM table — full width with repeating headers on page breaks */
+            .vlsm-results { overflow-x: visible; }
+            .vlsm-table { width: 100%; table-layout: fixed; word-break: break-word; }
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)